pvmismatch/pvmismatch_lib/pvsystem.py

```diff
@@ -94,9 +94,21 @@ def calcSystem(self):
 
     def calcMPP_IscVocFFeff(self):
         mpp = np.argmax(self.Psys)
-        Pmp = self.Psys[mpp]
-        Vmp = self.Vsys[mpp]
-        Imp = self.Isys[mpp]
+        P = self.Psys[mpp - 1:mpp + 2]
+        V = self.Vsys[mpp - 1:mpp + 2]
+        I = self.Isys[mpp - 1:mpp + 2]

```

Comment:
Why `+2`, I don't think you end up using `P[3]`, `V[3]`, or `I[3]` anywhere do you? I think just `+1` is enough, right?

    P = self.Psys[mpp - 1:mpp + 2]

gives you `P[0]` equivalent to `self.Psys[mpp-1]`, `P[1]` at `self.Psys[mpp]`, and finally `P[2]` for `self.Psys[mpp+1]`

Reply:
This is just a half-open interval issue, if we need the size of dP and dV to be (2, 1), we need three points, which this indexing provides. +1 just provides [mpp-1, mpp], rather than the intended [mpp-1, mpp, mpp+1]

Reply:
Oh, you are soooo right! Thanks! Crazy Python!